lib/options.ts

```diff
@@ -51,6 +51,8 @@ export type UploadInput =
 export interface UploadOptions {
   endpoint?: string
 
+  projectId?: string
```

Comment:
Can you explain what this option is about?

Reply:
To allow users to differentiate the same file across multiple projects, they could append a project ID to the upload so that fingerprints of the same file in different projects don't match. As you described it [here](https://github.com/tus/tus-js-client/issues/738#issuecomment-2841200871).

Does this align with your idea, or do you envision a different approach?

Reply:
This field has been removed.